### PR TITLE
add `npu_group_size` for `transformers_int4_npu_win` in all-in-one benchmark api

### PR DESCRIPTION
**Pull request by ch1y0q**
small bugfix

### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -2193,6 +2198,8 @@ def run_pipeline_parallel_gpu(repo_id,
         optimize_model = conf['optimize_model']
     if 'group_size' in conf:
         group_size = conf['group_size']
```
Comment (rnwang04) [inline python/llm/dev/benchmark/all-in-one/run.py:2200]: remove above if.

**Review by rnwang04 — APPROVED** [review]
others LGTM.

**Review by cyita — APPROVED** [review]

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 2]
```diff
@@ -214,7 +214,8 @@ def run_model(repo_id, test_api, in_out_pairs, local_model_hub=None, warm_up=1,
                             round(result[in_out_pair][-1][5], 2),
                             result[in_out_pair][-1][6] if any(keyword in test_api for keyword in ['int4_gpu', 'int4_fp16_gpu_win', 'int4_loadlowbit_gpu', 'int4_fp16_loadlowbit_gpu', 'fp16_gpu', 'deepspeed_optimize_model_gpu']) and not lookahead else 'N/A',
                             streaming if 'win' in test_api else 'N/A',
-                            use_fp16_torch_dtype if 'pipeline_parallel_gpu' in test_api else 'N/A'],
+                            use_fp16_torch_dtype if 'pipeline_parallel_gpu' in test_api else 'N/A',
+                            group_size],
```
Comment (Oscilloscope98) [inline python/llm/dev/benchmark/all-in-one/run.py:218]: Should we make group_size value "N/A" for other test_api? Maybe confusing

Comment (rnwang04) [reply python/llm/dev/benchmark/all-in-one/run.py:218]: Good point, we will fix this in next PR . 😊

**Pull request merged**